### PR TITLE
Updated maven project structure

### PR DESCRIPTION
**Pull request by ChrisMcCat**
Changed java from 11 to 13

### CODE REVIEW

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

File: src/main/java/com/infoshareacademy/App.java [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-        System.out.println( "Hello World!" );
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class App {
+
+    private static final Logger stdout = LoggerFactory.getLogger("CONSOLE_OUT");
+
+    public static void main( String[] args ) {
+        stdout.info( "Hello World!" );
```
Comment (rafalmisiak) [inline src/main/java/com/infoshareacademy/App.java]: a te spacje po co ? :)

File: pom.xml [hunk 2]
```diff
@@ -12,20 +12,24 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <maven.compiler.source>11</maven.compiler.source>
-    <maven.compiler.target>11</maven.compiler.target>
+    <maven.compiler.source>13</maven.compiler.source>
```
Comment (rafalmisiak) [inline pom.xml]: musze Was zmartić, jenkins nie jest jednak podbity :/ musimy wrócić do 11

Comment (ChrisMcCat) [reply pom.xml]: :(

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

File: .gitignore [hunk 3]
```diff
@@ -26,3 +26,4 @@ hs_err_pid*
 *.iml
 out
 gen
+target/
```
Comment (GregChyla) [inline .gitignore]: Suggestion to add .idea

File: pom.xml [hunk 4]
```diff
@@ -18,14 +18,18 @@
...
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <version>1.7.29</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-slf4j-impl</artifactId>
+      <version>2.12.1</version>
     </dependency>
   </dependencies>
```
Comment (GregChyla) [inline pom.xml]: delete

File: pom.xml [hunk 5]
```diff
@@ -18,14 +18,18 @@
...
 
   <build>
-
```
Comment (GregChyla) [inline pom.xml]: delete

File: pom.xml [hunk 6]
```diff
@@ -18,14 +18,18 @@
 
   <dependencies>
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>4.11</version>
```
Comment (GregChyla) [inline pom.xml]: change to Junit 5 engine

File: pom.xml (hunk 4 above)
Comment (rafalmisiak) [reply pom.xml]: dlaczego? To są loggery o których dzisiaj rozmawialismy

File: pom.xml (hunk 6 above)
Comment (rafalmisiak) [reply pom.xml]: junit5 dodacie na zajęciach z junita, póki co wywalić całkowicie junit

File: .gitignore (hunk 3 above)
Comment (ChrisMcCat) [reply .gitignore]: done already in previous commits

File: pom.xml (hunk 4 above)
Comment (ChrisMcCat) [reply pom.xml]: logger for sonar

File: pom.xml (hunk 5 above)
Comment (ChrisMcCat) [reply pom.xml]: done already in previous commits

File: pom.xml (hunk 6 above)
Comment (ChrisMcCat) [reply pom.xml]: cutted out for now

**Pull request merged**